designer/client/__tests__/page-create.jest.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { initI18n } from "../i18n";
+import { render } from "@testing-library/react";
+import PageCreate from "../page-create";
+
+describe("page create", () => {
+  initI18n();
+
+  describe("Page create fields fields", () => {
+    let props;
+    let textFieldEditPage;
+
+    beforeEach(() => {
+      props = {
+        path: "/some-path",
+        data: {
+          sections: [],
+          pages: [],
+        },
+      };
+
+      textFieldEditPage = render(<PageCreate {...props} />);
```

Comment:
this beforeEach block is redundant - it doesn't seem like you are changing props or have a need to reset them.

```suggestion
const { getByText } = render(<PageCreate {...props} />);
```
using this will clean up your `textFieldEditPage.getByText` to just `getByText` - and you will just be rendering once.

Reply:
The props I agree with. I think the render doesn't render outside of either a before or a test which is why the before was there. Do you have a suggestion for an alternative as I don't really want to replicate it in all the tests?